CHANGELOG.MD

```diff
@@ -8,7 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- Nothing
+- More template information to the README ([#22](https://github.com/stactools-packages/template/pull/22))
```

Comment:
I wiped out the changelog yesterday. Should we be leaving it empty, or should we keep it updated in the template repo and add instructions to empty it out when the template is used?

Reply:
I wasn't sure either, but thinking about it keeping it blank seems like a good option. I'll make that change.